docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,330 @@
-information about the FV3 dynamical core can be found `here 
-<https://noaa-emc.github.io/FV3_Dycore_ufs-v2.0.0/html/index.html>`_ and on the `NOAA Geophysical
-Fluid Dynamics Laboratory website <https://www.gfdl.noaa.gov/fv3/>`_.
-
-Interoperable atmospheric physics, along with the Noah Multi-parameterization (Noah MP)
-Land Surface Model options, are supported through the Common Community Physics Package
-(:term:`CCPP`; described `here <https://dtcenter.org/community-code/common-community-physics-package-ccpp>`_).
-Atmospheric physics are a set of numerical methods describing small-scale processes such
-as clouds, turbulence, radiation, and their interactions. There are two physics options
-supported for the release. The first is an experimental physics suite being tested for use
-in the future operational implementation of the Rapid Refresh Forecast System (RRFS) planned
-for 2023-2024, and the second is an updated version of the physics suite used in the operational
-Global Forecast System (GFS) v15. A scientific description of the CCPP parameterizations and
-suites can be found in the `CCPP Scientific Documentation <https://dtcenter.ucar.edu/GMTB/v5.0.0/sci_doc/index.html>`_,
-and CCPP technical aspects are described in the `CCPP Technical Documentation
-<https://ccpp-techdoc.readthedocs.io/en/v5.0.0/>`_. The model namelist has many settings
-beyond the physics options that can optimize various aspects of the model for use with each
-of the supported suites. 
-
-The SRW App supports the use of both GRIB2 and :term:`NEMSIO` input data. The UFS Weather Model
-ingests initial and lateral boundary condition files produced by :term:`chgres_cube` and outputs files in
-netCDF format on a specific projection (e.g., Lambert Conformal) in the horizontal and model
-levels in the vertical.
-
-Post-processor
-==============
+(:term:`FV3`) dynamical core configured with a Limited Area Model (LAM) capability (:cite:t:`BlackEtAl2020`).
+The dynamical core is the computational part of a model that solves the equations of fluid motion. A User’s Guide for the UFS :term:`Weather Model` can be found `here <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`__. 
+
+Common Community Physics Package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `Common Community Physics Package <https://dtcenter.org/community-code/common-community-physics-package-ccpp>`_ (:term:`CCPP`) supports interoperable atmospheric physics and Noah Multi-parameterization (Noah MP) Land Surface Model options. Atmospheric physics are a set of numerical methods describing small-scale processes such as clouds, turbulence, radiation, and their interactions. The SRW release includes an experimental physics version and an updated operational version. 
+
+Data Format
+^^^^^^^^^^^^^^^^^^^^^^
+
+The SRW App supports the use of both :term:`GRIB2` and :term:`NEMSIO` input data. The UFS Weather Model ingests initial and lateral boundary condition files produced by :term:`chgres_cube`. 
+
```

Comment:
I might say "The SRW App supports the use of external model data in GRIB2, netCDF, and nemsio format when generating initial and boundary conditions."  (The SRW App also supports netCDF files for global model data).